float32 motor_speed_now
float32 motor_angle_now
float32 motor_heat
float32 current
float32 voltage